interactive_marker_tutorials/package.xml

```diff
@@ -23,6 +23,15 @@
   <depend>tf2_ros</depend>
   <depend>visualization_msgs</depend>
 
+  <exec_depend>rclpy</exec_depend>
+  <exec_depend>rosidl_runtime_py</exec_depend>
+
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_cmake_flake8</test_depend>
+  <test_depend>ament_cmake_pep257</test_depend>
+  <!--TODO(jacobperron): add linter tests for C++ -->
```

Comment:
Should this be resolved now or later?

Reply:
In the past.
Since this PR is only about the Python code, I'd rather lint the C++ code in a follow-up (there will be errors to fix).

Reply:
Follow-up for adding C++ linter tests https://github.com/ros-visualization/visualization_tutorials/pull/55